lib/hooks/create/establishment.js

```diff
@@ -26,6 +26,16 @@ module.exports = settings => {
               return model.setStatus(withLicensing.id);
             }
           });
+      case 'update':
+        return Promise.resolve()
+          .then(() => Profile.query().findById(changedBy))
+          .then(profile => {
+            if (profile.asruLicensing) {
```

Comment:
This should be `profile.asruUser && profile.asruLicensing`. It's a tiny edge case, but it's real.

```suggestion
            if (profile.asruUser && profile.asruLicensing) {
```

Reply:
done